CONFIGURING_ANALYZERS.md

```diff
@@ -0,0 +1,134 @@
+# Configuring Analyzers
+
+There are many other analyzers included in Lucene and there are occasions where custom analyzers are needed. 
+There is now support for configuring additional analyzers without needing to further modify couchdb-lucene. 
+This extension is a loose adaptation of the [eXist-db/Lucene integration](https://github.com/eXist-db/exist/blob/develop/extensions/indexes/lucene/src/org/exist/indexing/lucene/AnalyzerConfig.java)
```

Comment:
I'm not sure what the test for "derivative work" is. I was aware of the functionality from having worked with eXist-db and intent on developing similar functionality in the context of couchdb-lucene. The core of the functionality is the use of native java reflection which then needed to be fit into the couchdb-lucene configuration and instantiation framework already in place.

I think the current [PR #249](https://github.com/rnewson/couchdb-lucene/pull/249) is largely independent of the [eXist-db code](https://github.com/eXist-db/exist/blob/develop/extensions/indexes/lucene/src/org/exist/indexing/lucene/AnalyzerConfig.java) and overlaps to the extent that there are limited ways in which java reflection is used to fetch a class constructor and invoke it.